engine/class_modules/priest/sc_priest.hpp

```diff
@@ -986,6 +1016,10 @@ struct priest_action_t : public Base
         {priest().specs.shadow_priest->effectN( 2 ),      affected_by.shadow_priest_ta},
         {priest().specs.holy_priest->effectN( 3 ),        affected_by.holy_priest_da},
         {priest().specs.holy_priest->effectN( 4 ),        affected_by.holy_priest_ta},
+        {priest().specs.discipline_priest->effectN( 1 ),  affected_by.discipline_priest_da},
+        {priest().specs.discipline_priest->effectN( 2 ),  affected_by.discipline_priest_ta},
+        {priest().talents.sins_of_the_many->effectN( 1), affected_by.sins_of_the_many_da},
+        //{priest().buffs.sins_of_the_many->data().effectN( 1), affected_by.sins_of_the_many_ta },
```

Comment:
can you add a comment for why this is still left in commented in? Just so that folks know why its not being used